spec/helpers/programme_activity_grouping_helper_spec.rb

```diff
@@ -0,0 +1,76 @@
+require "rails_helper"
+
+RSpec.describe ProgrammeActivityGroupingHelper do
+  describe "#primary_completion_instruction" do
+    context "when web_copy_completion_instruction is present" do
+      let(:group) { create(:programme_activity_grouping, web_copy_completion_instruction:) }
+      let(:web_copy_completion_instruction) { "enable you to develop your teaching expertise" }
```

Comment:
This is just super picky now, but I think it would just look better if let(:web_copy_completion_instruction) was above the let(:group), as I was confused for a second where it was getting the value from. (Same for the next few tests too)

Doesn't make any difference to the code, thats just my personal preference, so will leave it up to you 😄